.github/workflows/main.yml

```diff
@@ -0,0 +1,94 @@
+name: Godot CI
+
+on:
+    pull_request:
+    push:
+      branches: [ master, 3.2, 3.1, 3.0 ]
+env:
+  SCONS_CACHE_MSVC_CONFIG: true
+  SCONS_CACHE_LIMIT: 2048
+jobs:
+  windows-editor:
+    runs-on: "windows-latest"
+    name: Windows Editor (release_debug)
+    steps:
+    - uses: actions/checkout@v2 
+    # cache must be put in the github workspace directory on windows to be persisted correctly
+    - name: "force cache path to be correct"
+      run: echo "::set-env name=SCONS_CACHE::${{env.HOME}}/.scons_cache/"
+    # upload cache on completion and check it out now
+    - name: configure .scons_cache and download cache
+      id: scons-cache
+      uses: RevoluPowered/cache@v2.1
+      with:
+        path: ${{env.SCONS_CACHE}}
+        key: ${{ runner.os }}-godot-release_debug_editor
+        restore-keys: ${{ runner.os }}-godot-release_debug_editor
+    # use python 3.x release (works cross platform)
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v2
+      with:
+        # Semantic version range syntax or exact version of a Python version
+        python-version: '3.x' 
+        # Optional - x64 or x86 architecture, defaults to x64
+        architecture: 'x64' 
+    # You can test your matrix by printing the current Python version
+    - name: Display Python version
+      run: python -c "import sys; print(sys.version)"
+    - name: Installing scons and pywin32
```

Comment:
Does it still make a difference to use pywin32? AFAIK we no longer document the need to install it (CC @Calinou).

Reply:
As far as I know, it shouldn't be required anymore with recent Python versions, even if SCons warns you about it.

Reply:
It's still requested, so someone should look at this